index.js

```diff
@@ -0,0 +1,29 @@
+// this pulls the code from equal.js and renames it to run
```

Comment:
I know the slides reference `var`.  When we see them though, we should replace them with `let`. `var` can safely never be used in nodejs.